src/compiler/program.ts

```diff
@@ -226,6 +226,11 @@ namespace ts {
         for (const diagnostic of diagnostics) {
             output += formatDiagnostic(diagnostic, host);
         }
+
+        output += host.getNewLine();
+        output += `Compilation finished - ${diagnostics.length} issue${diagnostics.length > 1 ? "s" : ""}`;

```

Comment:
Instead of "issue" I would use the word "error" instead.
Error is the term used by TypeScript

Reply:
Gotcha. I used "issue" specifically because the `diagnostics` array includes errors, warnings, suggestions, etc. I see two options:

- Use a generic term (like "issues") and just give the length of the `diagnostics` array.
- Filter the `diagnostics` array based on types and report individual types of errors, warnings, suggestions, etc. 

What do you think?

Reply:
I though tsc was only returning "errors". Do we have examples of "warnings", "suggestions"...?
Returning just the total number of "issues" sounds fine because it's a summary not a detailed report.